hash-table/chaining.js

```diff
@@ -0,0 +1,139 @@
+const { Node, DoublyLinkedList } = require('../linked-list/doubly-linked-list')
+
+class ChainingHashTable {
+  constructor(capacity = 8) {
+    this.arr = new Array(capacity)
```

Comment:
What if I pass negative integers, or float numbers to this constructor? Or someone might pass a string as well (JS disadvantage, maybe)?